requirements/uitest.txt

```diff
@@ -0,0 +1,10 @@
+execnet==1.4.1
+fxapom==1.7.1
+py==1.4.31
+pyjwt==1.4.0
+PyPOM==1.0
+pytest==2.9.1
+pytest-instafail==0.3.0
+pytest-selenium==1.2.1
+pytest-variables==1.4
+pytest-xdist==1.14
```

Comment:
After we simplified our requirements files this should probably move to `requirements/dev.txt`